src/site/markdown/server/manage-guice-distributed-james.md

```diff
@@ -260,3 +261,102 @@ by setting the parameter `elasticsearch.index.mailbox.name` to the name of your
 re-creates index upon restart
 
 _Note_: keep in mind that reindexing can be a very long operation depending on the volume of mails you have stored.
+
+## Setting Cassandra user permissions
+
+With a Cassandra server that serving multiple applications including James. 
+The keyspace of each James server need to be protected from another, 
```

Comment:
a keyspace is not protected from another keyspace, it's protected from another James instance. We can say that keyspaces need isolation.